crates/aptos/src/genesis/keys.rs

```diff
@@ -313,12 +313,14 @@ impl CliCommand<()> for GenerateAdminWriteSet {
 
     async fn execute(self) -> CliTypedResult<()> {
         check_if_file_exists(self.output_file.as_path(), self.prompt_options)?;
-        let (bytecode, _script_hash) = self.compile_proposal_args.compile(self.prompt_options)?;
+        let (bytecode, _script_hash) = self
+            .compile_proposal_args
+            .compile("GenerateAdminWriteSet", self.prompt_options)?;
 
         let txn = Transaction::GenesisTransaction(WriteSetPayload::Script {
             execute_as: self.execute_as,
             script: Script::new(bytecode, vec![], vec![]),
-        });
+        })
```

Comment:
This probably will make it not compile

Reply:
I didn't delete this. Linter did :|

Reply:
Could have been my IDE too. Any way I added it back